documentation/asciidoc/accessories/build-hat/compat.adoc

```diff
@@ -1,48 +1,46 @@
 == Device Compatibility
 
-The Build HAT library supports all the LEGO® Technic™ devices included in the SPIKE™ Portfolio, along with those from the LEGO® Mindstorms Robot Inventor kit and other devices that use an LPF2 connector.
+The Build HAT library supports all the LEGO® Technic™ devices included in the SPIKE™ Portfolio, along with those from the LEGO® Mindstorms Robot Inventor kit and other devices that use an PoweredUp connector.
```

Comment:
nitpick: "use an PoweredUp" -> "use a PoweredUp" ?

Reply:
We probably need to add a note about LPF2 in somewhere. As far as I can see this is now the first place it's called PowerUp.

Reply:
LPF2 is an internal name only - PoweredUp is the correct term we should be using.